src/Files/MultilingualResources/Files.ru-RU.xlf

```diff
@@ -3223,67 +3235,67 @@ Files не собирает, хранит, распространяет или 
         </trans-unit>
         <trans-unit id="SettingsOpenFoldersWithOneClick" translate="yes" xml:space="preserve">
           <source>Open folders with a single click</source>
-          <target state="new">Open folders with a single click</target>
+          <target state="new">Открывать папки одним щелчком</target>
```

Comment:
Can you update the state to translated?

Reply:
Those. revert file with my commit?

Reply:
In order for the translation to work, you need to change `new` to `translated`. https://files.community/docs/contributing/translating-files

Reply:
Understood, I'm sorry, I didn't notice)

Reply:
Uploaded the changes